### PR TITLE
OOP fixes and command changes

### PR DESCRIPTION
**Pull request by jamesebond**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/DeleteStallReviewCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteStallReviewCommand}.
+ */
+public class DeleteStallReviewCommandTest {
```
Comment (teoks0199) [inline src/test/java/seedu/address/logic/commands/DeleteStallReviewCommandTest.java:22]: Good that theres some testing but can add more comprehensive testing next time too such as adding more tests in AddressBookParserTest

**Review by teoks0199 — DISMISSED** [review]
lgtm

File: src/test/java/seedu/address/logic/commands/DeleteStallReviewCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (beatricecst) [inline src/test/java/seedu/address/logic/commands/DeleteStallReviewCommandTest.java:23]: LGTM

**Review by beatricecst — APPROVED** [review]

**Review by teoks0199 — APPROVED** [review]
lgtm, dont forget to tag your issues too

File: src/main/java/seedu/address/logic/commands/AddStallReviewCommand.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
```
Comment (beatricecst) [inline src/main/java/seedu/address/logic/commands/AddStallReviewCommand.java:1]: LGTM

**Review by beatricecst — APPROVED** [review]

**Pull request merged**